drivers/ksfilter/swenum/swenum.rc

```diff
@@ -0,0 +1,5 @@
+#define REACTOS_VERSION_DLL
+#define REACTOS_STR_FILE_DESCRIPTION  "Plug and Play Software Device Enumerator"
+#define REACTOS_STR_INTERNAL_NAME     "swenum"
+#define REACTOS_STR_ORIGINAL_FILENAME "swenum.sys"
+#include <reactos/version.rc>
```

Comment:
Add a new empty line at the end, thanks!

Reply:
I add the newline at the end, and pushed the change. Hopefully you can see it, as the Github website doesn't seem to show it.

Reply:
I can see it's here (when there's no extra newline at the end, the GitHub website shows a "forbidden" panel icon. It doesn't show it now anymore, so it's good :)